src/infrastructure/middleware/authorization/whiteList/string.compare.ts

```diff
@@ -0,0 +1,22 @@
+import { EntriesUtil } from "./entriesUtil";
```

Comment:
This file name needs to be changed to StringCompare.ts.

Reply:
As the whiteList logic changed, this file was deleted as it was no longer used